.env-devel

```diff
@@ -223,6 +223,8 @@ DIRECTOR_V2_DYNAMIC_SCHEDULER_CLOSE_SERVICES_VIA_FRONTEND_WHEN_CREDITS_LIMIT_REA
 DIRECTOR_V2_SERVICES_CUSTOM_CONSTRAINTS=[]
 DIRECTOR_V2_GENERIC_RESOURCE_PLACEMENT_CONSTRAINTS_SUBSTITUTIONS='{}'
 DIRECTOR_V2_NODE_PORTS_STORAGE_AUTH=null
+DIRECTOR_V2_PUBLIC_API_BASE_URL=http://127.0.0.1:8006
```

Comment:
Maybe you can take the port from another env var?

Reply:
this is setup in the docker-compose ... it is possible to use yq for that but it is too convoluted provided that these ports never changed since the beginning and if they are changed it is easy to spot by searching the numbers in the rrpo.